Makefile

```diff
@@ -153,7 +153,7 @@ python-download-chain:
 swagger: config/swagger.yaml
 	@swagger-codegen generate -i $< -l erlang-server -o $(SWTEMP)
 	@echo "Swagger tempdir: $(SWTEMP)"
-	@cp $(SWTEMP)/priv/swagger.json $(HTTP_APP)/priv/
+	@cp $(SWTEMP)/priv/swagger.json $(HTTP_APP)/priv/swagger-docs
```

Comment:
swagger_router.erl appears to assume this location, so changing it may make starting node not possible (testing locally now).

Reply:
Confirmed - `make prod-build` succeeds then `make prod-start` fails.

Reply:
Copying swagger.json in both priv/ and priv/swagger-docs shall make the node working again (at the cost of some redundancy).